docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/BinaryClassification/Calibrators/Platt.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System;
+using System.Linq;
+
+namespace Microsoft.ML.Samples.Dynamic.Trainers.BinaryClassification.Calibrators
+{
+    public static class Platt
+    {
+        public static void Example()
+        {
+            // Create a new context for ML.NET operations. It can be used for exception tracking and logging, 
+            // as a catalog of available operations and as the source of randomness.
+            // Setting the seed to a fixed number in this example to make outputs deterministic.
+            var mlContext = new MLContext(seed: 0);
+
+            // Download and featurize the dataset.
+            var data = SamplesUtils.DatasetUtils.LoadFeaturizedAdultDataset(mlContext);
+            // Leave out 10% of data for testing.
+            var trainTestData = mlContext.BinaryClassification.TrainTestSplit(data, testFraction: 0.3);
+
+            // Create data training pipeline for non calibrated trainer and train Naive calibrator on top of it.
+            var pipeline = mlContext.BinaryClassification.Trainers.AveragedPerceptron();
+
+            // Fit the pipeline, and get a transformer that knows how to score new data.  
+            var transformer = pipeline.Fit(trainTestData.TrainSet);
+            // Fit this pipeline to the training data.
+            // Let's score the new data. The score will give us a numerical estimation of the chance that the particular sample 
+            // bears positive sentiment. This estimate is relative to the numbers obtained. 
+            var scoredData = transformer.Transform(trainTestData.TestSet);
+            var scoredDataPreview = scoredData.Preview();
+
+            PrintRowViewValues(scoredDataPreview);
+            // Preview of scoredDataPreview.RowView
+            // Score   4.18144
+            // Score  -14.10248
+            // Score   2.731951
+            // Score  -2.554229
+            // Score   5.36571
+
+            // Let's train a calibrator estimator on this scored dataset. The trained calibrator estimator produces a transformer
+            // that can transform the scored data by adding a new column names "Probability". 
+            var calibratorEstimator = mlContext.BinaryClassification.Calibrators.Platt();
+            var calibratorTransformer = calibratorEstimator.Fit(scoredData);
```

Comment:
>scoredData [](start = 64, length = 10)

we should probably train the calibrator on another dataset, not the testSet, in those samples. So have training set, testing set, and a calibrator training set